websites/R/Raven Code/presence.ts

```diff
@@ -0,0 +1,83 @@
+const presence = new Presence({ clientId: "937015924425367643" }),
+	startTimestamp = Math.floor(Date.now() / 1000);
+let languages: { key: string; page: string; text: string }[] = [];
+
+setInterval(
+	() =>
+		fetch("https://premid.ravencode.live").then(
+			async res => (languages = await res.json())
+		),
```

Comment:
Not sure if the parenthesis are necessary but this is better
```suggestion
	async () =>
		(languages = await fetch("https://premid.ravencode.live").then(res => res.json()))),
```

Reply:
prettier put the parenthesis but yes that's better